scripts/job.star

```diff
@@ -50,3 +53,17 @@ def scan_docker_image_high_critical_vulnerabilities_step(edition):
         'image': 'aquasec/trivy:0.18.3',
         'commands': commands,
     }
+
+def slack_job_failed_step(channel):
+    return {
+        'name': 'slack',
```

Comment:
A different name - like `notify_failure` should do be also fine?

Reply:
sure